File: knowledge-hub/knowledge-hub.WebAPI/Database/City.cs
using System.ComponentModel.DataAnnotations;

namespace knowledge_hub.WebAPI.Database
{
   public class City
   {
      [Key]
      public int CityId { get; set; }

      [Required]
      public string Name { get; set; }

      [Required]
      public string ZipCode { get; set; }

      [Required]
      public string Country { get; set; }
   }
}

File: knowledge-hub/knowledge-hub.Mobile/lib/models/city.dart
class City{
  int CityId = 1;
  String Name = "";
  String ZipCode = "";
  String Country = "";

  City();
  Map<String, dynamic> toJson() => {
    'cityId' : CityId,
    'name' : Name,
    'zipCode' : ZipCode,
    'country' : Country
  };
  City.fromJson(Map<String, dynamic> jsonFile):
        CityId = jsonFile['cityId']??"",
        Name = jsonFile['name']??"",
        ZipCode = jsonFile['zipCode']??"",
        Country = jsonFile['country']??"";
}